src/output.ts

```diff
@@ -0,0 +1,76 @@
+import { AfterStep, Before, BeforeStep, Status } from '@cucumber/cucumber';
+import {
+  ITestStepHookParameter,
+} from '@cucumber/cucumber/lib/support_code_library_builder/types';
+
+const NO_OUTPUT = '<Nothing was captured>';
+
+interface Writable {
+  write: (a: string) => void;
+}
+
+interface Interceptor {
+  capturedData: string[];
+  restore: () => void;
+  intercept: () => void;
+}
+
+const intercept = (
+  target: Writable,
+  capture?: boolean,
+  suppress?: boolean,
+): Interceptor => {
+  const capturedData: string[] = [];
+  const originalWrite = target.write;
+  return {
+    capturedData,
+    intercept: () => {
+      target.write = (a: string) => {
+        if (capture) {
+          capturedData.push(a);
+        }
+        if (!suppress) {
+          originalWrite.bind(target)(a);
```

Comment:
do `originalWrite.call(target, a)` instead to avoid creating a new function only for this execution

Reply:
Fixed